fluent-plugin-ddl-redis-store.gemspec

```diff
@@ -1,13 +1,13 @@
 # -*- encoding: utf-8 -*-
 Gem::Specification.new do |gem|
-    gem.name        = "fluent-plugin-redis-store"
-    gem.email       = "hanai@pokelabo.co.jp"
-    gem.version     = "0.2.0"
+    gem.name        = "fluent-plugin-ddl-redis-store"
+    gem.email       = "info@dominodatalab.com"
```

Comment:
@urianchang Do we have an email we can put here?

Reply:
`builds@dominodatalab.com`